### PR TITLE
[DNS] / [LP] email clarifications

### PR DESCRIPTION
**Pull request by kodster28**
Trying to make options simpler / more visible, a la [this community thread](https://community.cloudflare.com/t/what-are-the-smtp-settings-for-email-through-cloudflare/507257/4).

### CODE REVIEW

File: content/dns/_partials/_add-mx-records.md [hunk 1]
```diff
@@ -76,7 +78,7 @@ To route emails to your mail server, you need to [create two DNS records](/dns/m
       </div>
       </details>
 
-2.  An **MX** record that points to that subdomain.
+3.  [Add](/dns/manage-dns-records/how-to/create-dns-records/) an **MX** record that points to that subdomain.
```
Comment (pedrosousa) [inline content/dns/_partials/_add-mx-records.md]:
```suggestion
3.  [Add an `MX` record](/dns/manage-dns-records/how-to/create-dns-records/) that points to that subdomain.
```
Nit: The link in "Add" is probably too small and offers a small "click area".

File: content/dns/_partials/_add-mx-records.md [hunk 2]
```diff
@@ -5,15 +5,17 @@ _build:
...
-{{</Aside>}}
+- An SMTP provider.
+- To create two DNS records within Cloudflare.
 
-To route emails to your mail server, you need to [create two DNS records](/dns/manage-dns-records/how-to/create-dns-records/) within Cloudflare:
+To route emails through Cloudflare and to your mail server:
 
-1.  An **A** or **AAAA** record for your mail subdomain that points to the IP address of your mail server.
+1. Get the IP address and MX record details from your SMTP provider ([vendor-specific guidelines](/dns/manage-dns-records/reference/vendor-specific-records/)).
+2. [Add](/dns/manage-dns-records/how-to/create-dns-records/) an `A` or `AAAA` record for your mail subdomain that points to the IP address of your mail server.
```
Comment (pedrosousa) [inline content/dns/_partials/_add-mx-records.md]:
```suggestion
2. [Add an `A` or `AAAA` record](/dns/manage-dns-records/how-to/create-dns-records/) for your mail subdomain that points to the IP address of your mail server.
```

File: content/dns/_partials/_add-mx-records.md [hunk 3]
```diff
@@ -5,15 +5,17 @@ _build:
...
 
-If you are using [Email Routing](/email-routing/), you cannot modify the MX records for your Cloudflare zone without [disabling Email Routing](/email-routing/setup/disable-email-routing/).
+If you need to **send** emails from your domain, you need:
 
-{{</Aside>}}
+- An SMTP provider.
+- To create two DNS records within Cloudflare.
 
-To route emails to your mail server, you need to [create two DNS records](/dns/manage-dns-records/how-to/create-dns-records/) within Cloudflare:
+To route emails through Cloudflare and to your mail server:
```
Comment (pedrosousa) [inline content/dns/_partials/_add-mx-records.md:19]: Nit: I don't think it's entirely clear who should follow for these instructions (folks receiving emails, sending emails, or both?).

File: content/dns/manage-dns-records/how-to/email-records.md [hunk 4]
```diff
@@ -6,11 +6,11 @@ weight: 5
 
 # Set up email records
 
-There are two reasons to set up email records for your domain: to make sure email [reaches your mail server](#add-mx-records) and to prevent other email senders from [spoofing your domain](#prevent-domain-spoofing).
+There are two reasons to set up email records for your domain: to make sure email [reaches your mail server](#send-and-receive-email) and to prevent other email senders from [spoofing your domain](#prevent-domain-spoofing).
 
 ---
 
-## Add MX records
+## Send and receive email
```
Comment (pedrosousa) [inline content/dns/manage-dns-records/how-to/email-records.md]: Nit: There's a link to the old heading in:
* `content/dns/manage-dns-records/reference/dns-record-types.md`
* `content/load-balancing/additional-options/additional-dns-records.md`

**Review by pedrosousa — APPROVED** [review]

File: content/dns/_partials/_add-mx-records.md [hunk 5]
```diff
@@ -5,15 +5,17 @@ _build:
   list: never
 ---
 
-{{<Aside type="note">}}
+If you only need to **receive** emails, Cloudflare offers [Email Routing](/email-routing/) for free email forwarding to custom email addresses.
 
-If you are using [Email Routing](/email-routing/), you cannot modify the MX records for your Cloudflare zone without [disabling Email Routing](/email-routing/setup/disable-email-routing/).
+If you need to **send** emails from your domain, you need:
```
Comment (kodster28) [inline content/dns/_partials/_add-mx-records.md]:
```suggestion
To **send and receive** emails from your domain, you need:
```

File: content/dns/_partials/_add-mx-records.md (hunk 3 above)
Comment (kodster28) [reply content/dns/_partials/_add-mx-records.md:19]: Good feedback! Re-structured the partial to include headers separating these groups.

**Pull request merged**